src/Hug.js

```diff
@@ -1,32 +1,78 @@
 import React from 'react'
+import Box from './Box'
 import Card from './Card'
 import Flex from './Flex'
 import Icon from './Icon'
 import Text from './Text'
 import styled from 'styled-components'
+import { responsiveStyle } from 'styled-system'
 import PropTypes from 'prop-types'
 
-const StyledCard = styled(Card)`
-  border-radius: ${props => props.theme.radius};
+const display = responsiveStyle({
+  prop: 'display',
+  cssProperty: 'display'
+})
+
+const HugCard = styled(Card)`
+  border-top-left-radius: ${props => props.theme.radius};
+  border-top-right-radius: ${props => props.theme.radius};
+  border-bottom-right-radius: 0;
+  border-bottom-left-radius: 0;
+  overflow: hidden;
+`
+
+const BorderConcealer = styled(Box)`
+  z-index: -1;
+  position: relative;
```

Comment:
Not crazy about adding a new positioning context here, but with the negative margin approach I guess it's unavoidable. The border could probably be overridden with CSS as well, which would mean less logic in JS. I think `& > * { border: 0; }` should work

Reply:
These comments are probably better for #125 - would be helpful to use more descriptive PR titles like, *Adjust child border styles in Hug component*